Cargo.toml

```diff
@@ -22,3 +22,6 @@ flate2 = "1.0.14"
 mdcat = { version = "0.16.1", default-features = false }
 pulldown-cmark = "0.7.1"
 syntect = "4.1.1"
+serde = { version = "1.0", features = ['derive'] }
+ron = "0.5"
+anyhow = "1.0.31"
```

Comment:
Super cool. Never heard of this before. I'll replace all my boxing stuff once this is merged

Reply:
Yeah, `anyhow` is the bomb! If you want to derive custom error types very easily, you should also check out `thiserror`. Between those two, you should be set on error handling.